design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,212 @@
+type Mutation {
+    createNegotiableQuote(
+        cartId: ID! @doc(description: "Cart ID")
+        quoteName: String @doc(description: "Quote name")
+        comment: String @doc(description: "Comment")
+        files: [FileInput!] @doc(description: "Attached files")
+    ): CreateNegotiableQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    addNegotiableQuoteComments(
+        cartId: ID! @doc(description: "Cart ID")
+        comment: String @doc(description: "Comment")
+        files: [FileInput!] @doc(description: "Attached files")
+    ): AddNegotiableQuoteCommentsOutput! @doc(description:"Generates negotiable quote request")
+
+    updateNegotiableQuoteItemsQuantity(
+        cart_id: ID!, @doc(description: "unique Id of negotiable quote")
+        items: [ID!]! @doc(description: "unique Ids of Items to be update in negotiable quote")
+        quantity: Float, @doc(description: "quantity of the item  to be updated")
+    ): UpdateNegotiableQuoteItemsQuantityOutput @doc(description: "Remove Items in negotiable quote")
```

Comment:
Naming seems off here: the name of this mutation is `updateNegotiableQuoteItemsQuantity`, but the description is `Remove Items in negotiable quote`.